101-vm-ws2016-visualstudio2019/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "itemDisplayName": "Deploy a Windows SV VM with Visual Studio.",
+  "description": "This template deploys a Windows Server VM with Visual Code Studio Community 2019, with a few options of VM. For that you need to inform the name of VM, admin user and admin password.",
+  "summary": "This template deploys a Windows Server VM and also deploys Visual Studio Community 2019.",
+  "githubUsername": "Krisnatagoras",
+  "dateUpdated": "2019-10-31",
+  "type": "QuickStart",
+  "tags":[
```

Comment:
how are these used?

Reply:
ok, chatted w/ Lee - let's make this an object:

"tags": {
        "key": "the value for the tag"
},